### PR TITLE
Update the docs

### PR DESCRIPTION
**Pull request by Pepo48**
* updated the release notes
* updated the FIPS guide

### CODE REVIEW

**Review by vmuzikar — DISMISSED** [review]
@Pepo48 Thank you, LGTM!

Before merging, I'd like to ask also @pedroigor / @mposolda for a review as we're touching FIPS here.

**Review by andre-nascimento6791 — COMMENTED** [review]
The changes LGTM, @Pepo48 .

Good job.

**Comment by abstractj** [comment]
@Pepo48 could you please solve those conflicts?

**Comment by Pepo48** [comment]
> @Pepo48 could you please solve those conflicts?

@abstractj done.

**Review by vmuzikar — DISMISSED** [review]

File: docs/guides/server/fips.adoc [hunk 1]
```diff
@@ -35,7 +35,13 @@ BouncyCastle FIPS can be downloaded from the https://www.bouncycastle.org/fips-j
 
 You can create either `pkcs12` or `bcfks` keystore to be used for the Keycloak server SSL. The `pkcs12` works well in BCFIPS non-approved mode.
 
-PKCS12 keystore can be generated with OpenJDK 17 Java on RHEL 9 in the standard way.
+PKCS12 keystore can be generated with OpenJDK 17 Java on RHEL 9 in the standard way. Make sure that the system is in FIPS mode, you can check it with the following command:
+
+```bash
+fips-mode-setup --check
```
Comment (pedroigor) [inline docs/guides/server/fips.adoc:41]: Isn't  `fips-mode-setup --enabled` the command that actually changes the Java security files?

Comment (Pepo48) [reply docs/guides/server/fips.adoc:41]: Correct, originally there was `--enabled` param, then I realized that I meant `--check` param for checking. And I forgot to change the following sentence.

Let me fix it quickly.

**Review by andre-nascimento6791 — DISMISSED** [review]
LGTM, @Pepo48 .

**Comment by vmuzikar** [comment]
@Pepo48 Could you please rebase?

**Comment by abstractj** [comment]
@vmuzikar I believe it is time to merge. I'd argue that we need an issue ID for that, but I will leave that up to you.

**Review by pedroigor — APPROVED** [review]

**Comment by vmuzikar** [comment]
@abstractj If my understanding is correct, issue ID was [not necessary](https://github.com/keycloak/keycloak/blob/3b2dea64ac8c8bd224029a4d53a13d0d2cceb321/PR-CHECKLIST.md?plain=1#L7) as it was neither a new feature, nor a bug fix (affecting some release). :)

**Pull request merged**